RickAndMortyApp/Views/DetailsCharacterCollectionViewCell.swift

```diff
@@ -57,7 +57,7 @@ final class DetailsCharacterCollectionViewCell: UICollectionViewCell {
         }
     }
     
-    func setCell(viewModel: DetailsCharacterCellViewModel) {
+    func setCell(viewModel: LocationDetailsCharacterCellViewModel) {
         viewModel.registerForData { [weak self] data in
                     self?.imageView.af.setImage(withURL: URL(string: data.image) ?? URL(string: "https://rickandmortyapi.com/api/character/avatar/1.jpeg")!,
```

Comment:
You can use `URL(string: "https://rickandmortyapi.com/api/character/avatar/1.jpeg")!` as a constant. You are using some other cells and you can also use constant there.

Reply:
Done!